docs/getting-started/getting-started.md

```diff
@@ -47,12 +47,12 @@ With the remote helper installed, you will be able to [push](using-radicle/pushi
 
 ## Joining a seed node
 
-Seed nodes are "always-on" nodes running on public IPs that serve data to any connected peers. By joining a seed node, it automatically tracks you and shares your data across it's network of other connected users. This increases the availability of your data throughout the network, while making it easier to find other's data as well. We recommend joining the official Radicle seed node to bootstrap your connectivity.
+Seed nodes are "always-on" nodes running on public IPs that serve data to any connected peers. By joining a seed node, it automatically tracks you and shares your data across its network of other connected users. This increases the availability of your data throughout the network, while making it easier to find other's data as well. Upstream is preconfigured with an official Radicle seed node to bootstrap your connectivity. If you have removed the default seed node, you can always re-add it later by following these steps:
```

Comment:
```suggestion
Seed nodes are "always-on" nodes running on public IP addresses that serve data to any connected peers. By joining a seed node, it automatically tracks you and shares your data across its network of other connected users. This increases the availability of your data throughout the network, while making it easier to find other's data as well. Upstream is preconfigured with an official Radicle seed node to bootstrap your connectivity. If you have removed the default seed node, you can always re-add it later by following these steps:
```

Reply:
There's no such thing as "joining" a seed node at the moment.

Reply:
Should we rename this section to "Adding a seed node to Upstream"?

Reply:
We have _"Enter seed addresses that you’d like to subscribe to here."_ in Settings page of app.

Is this not accurate @cloudhead ?

Reply:
Yeah that's accurate.

Reply:
https://github.com/radicle-dev/radicle-docs/pull/13/commits/db26aca5e425dc27e50a95e8df5463900c6df788